tests/test_lcss_compress.py

```diff
@@ -0,0 +1,104 @@
+from unittest import TestCase
+
+from mappymatch.matchers.lcss.ops import compress
```

Comment:
I got an error on this line; should we be importing compress from the `mappymatch.matchers.lcss.utils` module instead?

Reply:
You are right. I didn't notice that the import error on the test in the vscode testing framework puts them into a different location. The bug has been fixed in the latest commit.